src/herder/TxSetFrame.h

```diff
@@ -22,12 +25,21 @@ class TxSetFrame
 
     Hash mPreviousLedgerHash;
 
-    bool
-    checkOrTrim(Application& app,
-                std::function<bool(TransactionFramePtr, SequenceNumber)>
-                    processInvalidTxLambda,
-                std::function<bool(std::vector<TransactionFramePtr> const&)>
-                    processLastInvalidTxLambda);
+    struct AccountTransactionQueue
+    {
+        AccountID mID;
```

Comment:
this member is not used, to it could be just `using AccountTransactionQueue = deque`

Reply:
good catch